apps/chatbot/src/modules/vector_database.py

```diff
@@ -43,17 +45,24 @@
 REDIS_URL = os.getenv('CHB_REDIS_URL')
 WEBSITE_URL = os.getenv('CHB_WEBSITE_URL')
 REDIS_CLIENT = Redis.from_url(REDIS_URL, socket_timeout=10)
-
+REDIS_ASYNC_CLIENT = aredis.Redis.from_pool(
+    aredis.ConnectionPool.from_url(REDIS_URL)
+)
+REDIS_INDEX_NAME = os.getenv('CHB_REDIS_INDEX_NAME')
 REDIS_SCHEMA = IndexSchema.from_dict({
-    "index": {"name": "index", "prefix": "index/vector"},
+    "index": {"name": REDIS_INDEX_NAME, "prefix": "index/vector"},
     "fields": [
         {"name": "id", "type": "tag", "attrs": {"sortable": False}},
         {"name": "doc_id", "type": "tag", "attrs": {"sortable": False}},
         {"name": "text", "type": "text", "attrs": {"weight": 1.0}},
-        {"name": "vector", "type": "vector", "attrs": {"dims": 1024, "algorithm": "flat", "distance_metric": "cosine"}}
+        {"name": "vector", "type": "vector", "attrs": {
+            "dims": 768,
```

Comment:
If you agree, I would make the `dims` value dynamic according to the selected embedding model